### DISCUSSION TITLE
Share workflows

### DISCUSSION THREAD

**Discussion by dvXcode** [drivardxrm/Dataverse-CustomApis#5] [General]
It will be nice if you could share all workflows that are used in this project.

**Comment by drivardxrm** [comment]
Hi @dvXcode,
I'm not sure I understand your request.
What workflows are you refering to?

This is a Dataverse Custom API collection and all the source code is available in the repository.

You can install the solution found in the release section and all the API will be available in your Dataverse environment.

**Comment by dvXcode** [comment]
Hi @drivardxrm 

Can you explain how you generate a solution in the "Driv.CustomApis.Solution" folder? I'm curious if you use any GitHub Actions workflows to automate this task.
![image](https://user-images.githubusercontent.com/51912361/221425670-c944ffbe-4ff7-4bea-aa81-be7b688c9302.png)

**Comment by drivardxrm** [comment]
Hi @dvXcode,

Actually I extracted the solution manualy using the script contained here https://github.com/drivardxrm/Dataverse-CustomApis/tree/main/Driv.CustomApis.Solution/spkl
and commited to the repo afterward

I also have an Azure Devops CI/CD pipeline linked to the repo that repackages the solution.
